### PR TITLE
chore: generate openapi specs for organization endpoint for v1 and v2

### PR DESCRIPTION
**Pull request by hrithikesh026**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement
- [x] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Description

generated openapi specs for organization endpoint for v1 and v2

### Additional Changes

- [ ] This PR modifies the API contract
- [ ] This PR modifies the database schema
- [ ] This PR modifies application configuration/environment variables

## How did you test it?

verified that the generated open_api_spec file is valid by running 
`swagger-cli validate ./api-reference/openapi_spec.json`

Checked if new endpoints are being rendered in mintlify
v2
Screenshot 2024-08-20 at 5 22 03 PM

v1
Screenshot 2024-08-20 at 5 16 43 PM

## Checklist

- [x] I formatted the code `cargo +nightly fmt --all`
- [x] I addressed lints thrown by `cargo clippy`
- [x] I reviewed the submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

File: crates/openapi/src/routes/organization.rs [hunk 1]
```diff
@@ -24,28 +25,30 @@
...
+    get,
     path = "/organization/{organization_id}",
     params (("organization_id" = String, Path, description = "The unique identifier for the Organization")),
     responses(
         (status = 200, description = "Organization Created", body =OrganizationResponse),
         (status = 400, description = "Invalid data")
     ),
     tag = "Organization",
-    operation_id = "Create an Organization",
+    operation_id = "Retrive an Organization",
```
Comment (SanchithHegde) [inline crates/openapi/src/routes/organization.rs:41]:
```suggestion
    operation_id = "Retrieve an Organization",
```

Please re-generate the OpenAPI spec once you make this change.

**Review by SanchithHegde — DISMISSED** [review]

**Review by Narayanbhat166 — COMMENTED** [review]
can you try to render the openapi locally and check if the endpoints are rendered?

You can follow the steps mentioned here https://mintlify.com/docs/development

File: crates/openapi/src/openapi_v2.rs [hunk 2]
```diff
@@ -69,6 +69,11 @@ Never share your secret api keys. Keep them guarded and secure.
     ),
     // The paths will be displayed in the same order as they are registered here
     paths(
+        // Routes for Organization
+        routes::organization::organization_create,
```
Comment (Narayanbhat166) [inline crates/openapi/src/openapi_v2.rs:73]: in order to add these endpoints to mintlify you will have to run this command, run it separately for both v1 and v2 folders

npx @mintlify/scraping@latest openapi-file openapi_spec.json  -o api-reference

**Review by SanchithHegde — APPROVED** [review]

**Review by Narayanbhat166 — APPROVED** [review]

File: api-reference-v2/api-reference/organization/organization--create.mdx [hunk 3]
```diff
@@ -0,0 +1,3 @@
+---
+openapi: post /organization
```
Comment (jarnura) [inline api-reference-v2/api-reference/organization/organization--create.mdx:2]: V2 will have v2 prefix

Comment (jarnura) [reply api-reference-v2/api-reference/organization/organization--create.mdx:2]: In other places too

**Review by jarnura — CHANGES_REQUESTED** [review]

File: api-reference-v2/api-reference/organization/organization--create.mdx (hunk 3 above)
Comment (hrithikesh026) [reply api-reference-v2/api-reference/organization/organization--create.mdx:2]: /organization can be considered as a new endpoint introduced in V2.

**Review by jarnura — APPROVED** [review]

**Review by Narayanbhat166 — APPROVED** [review]

**Pull request merged**